lib/resemble.js

```diff
@@ -383,7 +383,7 @@ function resembleJS(fileOne, fileTwo, options) {
 
         });
 
-        data.misMatchPercentage = (mismatchCount / (height * width) * 100).toFixed(2);
+        data.misMatchPercentage = (mismatchCount / (height * width) * 100).toFixed(5);
```

Comment:
Hmm, that was easy. On the other hand, as you say, this is a temp fix, it's still strange why this does work in the original resemble code with `toFixed(2)`. I'll merge it and make a bugifx release. Tnx man!!

Reply:
I didn't have enough time to try new version of resemble but I'm going to investigate it)

Reply:
I'm also busy with it. We can't copy it 1 on 1, because I've implemented multiple blockouts and so on.

Reply:
Yeah, I mentioned it) So, as soon as I'll try new version of resemble, I'll notify you about results)